agentos/cli.py

```diff
@@ -162,114 +19,27 @@ def agentos_cmd():
     "-l",
     metavar="PACKAGE_LOCATION",
     type=click.Path(),
-    default="./.acr",
+    default="./.aos",
     help="Path to AgentOS Component Registry installation directory",
 )
 @click.option(
     "--agent-file",
```

Comment:
Have you tested the `install` CLI command from outside of a directory using the `--agent-file` command? I vaguely remember trying that at some point and it failing.

Reply:
The following works for me:

```
mkdir test_agent
cd test_agent/
agentos init .
cd ..
agentos install cartpole -d test_agent/.aos/ -f test_agent/agentos.ini
cd test_agent/
agentos run
```

I'm guessing the problem is you weren't specifying both the AOS directory with `-d` **and** the `agentos.ini` with `-f`.   For now, I'll just throw an error if you specify `-d` without `-f` or vice versa.  

We could just have the `agentos.ini` specify the AOS directory path but I was thinking that you would maybe want to install an agent multiple time in different directories using the same `agentos.ini` file.  Maybe that's too much future-proofing though.

Reply:
Ok, cool. I just tried your script in WSL on my Windows machine, and it works.